tests/kernels/test_marlin_gemm.py

```diff
@@ -42,11 +44,30 @@
 DTYPES = [torch.float16, torch.bfloat16]
 
 
+class MarlinWorkspace:
```

Comment:
Looks like MarlinWorkspace is duplicated in benchmark_marlin.py

Reply:
will move back into utils